src/api/app/components/workflow_run_detail_component.html.haml

```diff
@@ -1,33 +1,27 @@
 %ul.nav.nav-tabs#workflow-run-tabs{ role: 'tablist' }
   %li.nav-item{ role: 'presentation' }
-    %a.nav-link.active{ id: "request-tab#{id}", data: { toggle: 'tab' }, href: "#request-tab-content#{id}",
-                        role: 'tab', aria: { controls: "request-tab-content#{id}", selected: 'true' } }
-      Request
-  %li.nav-item{ role: 'presentation' }
-    %a.nav-link{ id: "response-tab#{id}", data: { toggle: 'tab' }, href: "#response-tab-content#{id}",
-                  role: 'tab', aria: { controls: "response-tab-content#{id}", selected: 'false' } }
-      Response
+    %a.nav-link.active{ id: "incoming-webhook-tab#{id}", data: { toggle: 'tab' }, href: "#incoming-webhook-tab-content#{id}",
+                        role: 'tab', aria: { controls: "incoming-webhook-tab-content#{id}", selected: 'true' } }
+      Webhook from #{scm_vendor}
   %li.nav-item{ role: 'presentation' }
     %a.nav-link{ id: "artifacts#{id}", data: { toggle: 'tab' }, href: "#artifacts-tab-content#{id}",
                  role: 'tab', aria: { controls: "artifacts-tab-content#{id}", selected: 'false' } }
       Artifacts
 .tab-content.p-3#workflow-run-tabs-content
-  .tab-pane.fade.show.active{ id: "request-tab-content#{id}", role: 'tabpanel',
-                              aria: { labelledby: "request-tab#{id}" } }
-    %h5 Request Headers
+  .tab-pane.fade.show.active{ id: "incoming-webhook-tab-content#{id}", role: 'tabpanel',
+                              aria: { labelledby: "incoming-webhook-tab#{id}" } }
+    %h5 Webhook URL
+    %pre.border.p-2#response-url
+      = response_url # Change this to incoming webhook url
```

Comment:
Do you want to remove `Change this to incoming webhook url`?

Reply:
True!